docs/caveats.md

```diff
@@ -200,13 +200,13 @@ Sadly, it's also **NOT** possible to use *VRRP* on a *Virtual Network* interface
 * Fortinet configuration templates set OSPF network type based on number of neighbors, not based on **ospf.network_type** link/interface parameter.
 
 (caveats-frr)=
-## FRR
+## FRRouting
 
 * Many FRR configuration templates are not idempotent -- you cannot run **netlab initial** multiple times. Non-idempotent templates include VLAN and VRF configurations.
 * VM version of FRR is an Ubuntu VM. The FRR package is downloaded and installed during the initial configuration process.
 * You can change the FRR default profile with the **netlab_frr_defaults** node parameter (`traditional` or `datacenter`, default is `datacenter`).
 * **netlab collect** downloads FRR configuration but not Linux interface configuration.
-* FRR containers need host kernel modules for MPLS forwarding. If your Ubuntu distribution does not include the MPLS drivers, do `sudo apt install linux-generic`.
+* FRR containers need host kernel modules for VRFs, MPLS, and VXLAN forwarding. If your Ubuntu distribution does not include the MPLS drivers, try installing them with `sudo apt install linux-generic`. You cannot load kernel modules in GitHub containers; VRF, MPLS, and VXLAN modules are thus not working in that environment.
```

Comment:
Proposal:
`FRR containers need host kernel modules for VRFs, MPLS, and VXLAN forwarding, if the related modules are used by netlab.`

Reply:
(just to be clear: the message that we want to pass is: if you don't use vrf or mpls or vxlan, this will run without the modules / in codespaces)